src/anndata_repr/hover_icons.js

```diff
@@ -12,6 +12,19 @@ for (let b of blocks) {
     b.addEventListener("mouseout", () => {
         change_opacity(blocks, 0, 1)
     })
+    b.addEventListener("click", () => {
+
+        const elements = document.getElementsByClassName('ad-section-summary-in')
```

Comment:
To ensure unique selector with multiple svgs in the document, the unique string on top is used (with[ js replace ](https://github.com/manzt/anndata-repr/blob/5a3ca5b845f8708f56b766bf96cb32408c07f494/src/anndata_repr/_create_icons.py#L115)on "__REPLACE_ME__")

Should we also do this for the 'ad-section-summary-in' divs [created](https://github.com/manzt/anndata-repr/blob/5a3ca5b845f8708f56b766bf96cb32408c07f494/src/anndata_repr/_formatting_html_xarray.py#L192) in the collapsible section?

Reply:
okay, updated such that there is one unique id generated for each ann data output. Then that id is propogated down to the click handlers.